DEVELOPMENT.md

```diff
@@ -0,0 +1,132 @@
+# Development
+
+* [Prerequisites](#prerequisites)
+* [Debug via Blink Developer Tools / node-inspector](#debug-via-blink-developer-tools--node-inspector)
+* [Config file options](#config-file-options)
+* [Feature flags](#feature-flags)
+  * [How to add a new flag](#how-to-add-a-new-flag)
+  * [How to configure feature flags](#how-to-configure-feature-flags)
+* [Run in the cloud](#run-in-the-cloud)
+  * [Initial setup (one time per developer machine)](#initial-setup-one-time-per-developer-machine)
+  * [Create/update AWS EC2 cluster](#createupdate-aws-ec2-cluster)
+* [Dive in development](#dive-in-development)
+
```

Comment:
@troggy - Might want to start with clone the repository? Just to be explicit is all.